<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WArmadiettoLav</class>
 <widget class="QWidget" name="WArmadiettoLav">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="antaDaviBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>80</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Anta Davi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allObjsBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>All Objects</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>80</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Bagno</string>
   </property>
  </widget>
  <widget class="QPushButton" name="antaAryBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>80</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Anta Ary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLavBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafLav</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
